mixer/pkg/runtime/testing/data/data.go

```diff
@@ -242,6 +242,86 @@ metadata:
   namespace: istio-system
 `
 
+// DynamicAdapter is the adapter generated from: https://github.com/istio/istio/wiki/Mixer-Out-Of-Process-Adapter-Walkthrough
+// with the template reportnothing
+var DynamicAdapter = `
+apiVersion: "config.istio.io/v1alpha2"
+kind: adapter
+metadata:
+  name: tgrpcadapter
+  namespace: istio-system
+spec:
+  description: 
+  session_based: false
+  templates:
+  - tdynreport
+  config: ` + dynamicAdapterConfig
```

Comment:
I don't think it's necessary to pull in a new template and adapter descriptor just for this test.. There are already several test descriptors in mixer codebase that could be reused here. I made a change to do it and reconcile the newly added test with an existed test: https://github.com/istio/istio/compare/release-1.1...bianpengyuan:dynamic-handler-update?expand=1 Let me if you are Ok for me to take over your PR. Thanks!

Reply:
It's just fine. You can take over it.
By the way, I'd like to know if it's certain that mixer will be implemented as an extension of envoy in the future. Is there a roadmap of mixer v2?

Reply:
Yes, mixer will be moved into proxy in roughly 1.3 time frame.